modules/drivers/druid/test/metabase/driver/druid/query_processor_test.clj

```diff
@@ -283,44 +283,45 @@
 
 (deftest native-query-test
   (mt/test-driver :druid
-    (is (= {:row_count 2
```

Comment:
For this and the subsequent tests, I've just changed `=` to `partial=` and adjusted the indentation.